accounting/src/main/java/io/skai/accounting/feign/clients/NotificationClient.java

```diff
@@ -1,14 +1,14 @@
 package io.skai.accounting.feign.clients;
 
 import feign.RequestLine;
-import io.skai.accounting.dto.notificationDto.NewOrderDto;
+import io.skai.accounting.dto.notification.OrderNotificationDto;
 import org.springframework.cloud.openfeign.FeignClient;
 
 @FeignClient(name = "notifications",
```

Comment:
Does it required? Rest communication between accounting and notification is deprecated in favor of the kafka.

Reply:
No, i wanted to delete it in new pr, because this became too large. I will delete this module in fix-commit

Reply:
+